contracts/MechMarketplace.sol

```diff
@@ -260,7 +255,7 @@ contract MechMarketplace is IErrorsMarketplace {
             }
 
             // Calculate request Id
-            requestIds[i] = getRequestId(msg.sender, requestDatas[i], nonce);
+            requestIds[i] = getRequestId(msg.sender, requestDatas[i], deliveryRate, nonce);
```

Comment:
Adding deliveryRate into the requestId is required as this is the part of the request the requester signs on, and it becomes the only verifiable part of hash when requesting off-chain